OrbitCore/UserDefinedCaptureData.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CORE_USER_DEFINED_CAPTURE_DATA_H_
+#define ORBIT_CORE_USER_DEFINED_CAPTURE_DATA_H_
+
+#include "OrbitClientData/FunctionInfoSet.h"
+
+class UserDefinedCaptureData {
```

Comment:
Add a comment

Reply:
Done.

Reply:
So far it seems that you did not pushed the changes yet.
I have a rather conceptual question on this class. Should (if so how) should it be made thread-safe?
There should be probably a statement about thread-safty in the comment here, as this is a topic of special interest in the area of capture data, data manager, and app.

Reply:
Yes, sorry, I haven't pushed the change yet. Good point on thread safety, will at least add a comment. Right now, the data here can only be updated by the main thread.

Reply:
Done.